packages/gestalt/src/Link/Link.js

```diff
@@ -6,9 +6,9 @@ import styles from './Link.css';
 
 type Props = {|
   children?: React.Node,
-  href: string,
+  href?: string,
```

Comment:
What's the case for having a link without `href`?

Reply:
@chrislloyd according to the existing `Link` [docs](https://pinterest.github.io/gestalt/#/Link), it would work like this:
![image](https://user-images.githubusercontent.com/7877010/40741135-841463fc-63ff-11e8-9254-f3eac65c584d.png)

Though this is still broken because browsers don't like `<a>` as a child of `<a>`. If we choose to support this we'll have to make the element into a `div` or `span` if `href` is `undefined` (I can make a followup PR). The other option is to continue making it required but I would need to remove that from the docs

Reply:
@christianvuerings ^ - I agree that I don't think making it optional is the best way to get React-Router support out of the box.